nautobot/users/factory.py

```diff
@@ -31,3 +50,16 @@ class Meta:
     last_login = factory.Faker("date_time", tzinfo=timezone.utc)
 
     # TODO config_data
+
+
+class SavedViewFactory(BaseModelFactory):
+    class Meta:
+        model = SavedView
+
+    name = factory.LazyAttributeSequence(lambda o, n: f"Sample {o.view} Saved View - {n + 1}")
+    owner = random_instance(User, allow_null=False)
+    view = factory.Faker(
+        "random_element",
+        elements=populate_saved_view_view_name(),
```

Comment:
This is still being called at declaration/import time. Probably need to do something with a lambda function and/or a `LazyAttribute`, maybe something like:

```py
class Meta:
    exclude = ("ct",)

ct = random_instance(
    lambda: ContentType.objects.filter(FeatureQuery("saved_views").get_query()), allow_null=False
)
view = factory.LazyAttribute(lambda o: f"{o.ct.app_label}:{o.ct.model}_list")
```